fdbserver/DataDistribution.actor.cpp

```diff
@@ -1977,17 +1982,19 @@ void runAuditStorage(Reference<DataDistributor> self,
 	    auditState.getType() != AuditType::ValidateStorageServerShard) {
 		throw not_implemented();
 	}
+	TraceEvent(SevDebug, "DDRunAuditStorage", self->ddId)
+	    .detail("AuditState", auditState.toString())
+	    .detail("Context", context);
 	ASSERT(auditState.id.isValid());
-	ASSERT(!auditState.range.empty());
+	if (auditState.range.empty()) {
```

Comment:
If the `range.empty()`, do we want to continue the audit, or shall we fail it?

Reply:
Fail it would be more clear.